app.js

```diff
@@ -26,122 +27,107 @@ mobLink.addEventListener('click', () => { topNav.style.display = 'none'; });
+    image: 'images/Snapshoot Portfolio.jpg',
+    technologies: ['HTML', 'CSS', 'Javascript'],
+    liveLink: 'https://example.com/project6',
+    sourceLink: 'https://github.com/example/project6',
+  },
+  {
+    name: 'Keeping track of hundreds  of components website',
+    name2: 'Multi Post Stories',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.',
+    image: 'images/Snapshoot Portfolio.jpg',
+    technologies: ['HTML', 'CSS', 'Javascript'],
+    liveLink: 'https://example.com/project7',
+    sourceLink: 'https://github.com/example/project7',
+  },
+];
 
-// const projectsContainer = document.getElementById('work-container');
-// const popup = document.getElementById('popup');
-// const closeBtn = document.getElementById('close-btn');
-// const popupTitle = document.getElementById('popup-title');
-// const popupDescription = document.getElementById('popup-description');
-// const popupImage = document.getElementById('popup-image');
-// const popupTechnologies = document.getElementById('popup-technologies');
-// const liveLink = document.getElementById('live-link');
-// const sourceLink = document.getElementById('source-link');
+const showCaseBtns = document.querySelectorAll('#showcase-button');
+const popup = document.getElementById('popup');
+const closeBtn = document.querySelector('.close-btn');
+const popupTitle = document.getElementById('popup-title');
+const popupTitle2 = document.getElementById('popup-title2');
+const popupDescription = document.getElementById('popup-description');
+const popupImage = document.getElementById('popup-image');
+const popupTechnologiesOne = document.getElementById('popup-technologies-one');
+const popupTechnologiesTwo = document.getElementById('popup-technologies-two');
+const popupTechnologiesThree = document.getElementById('popup-technologies-three');
+const liveLink = document.getElementById('live-link');
+const sourceLink = document.getElementById('source-link');
 
-// // // Function to create project cards dynamically
-// // function createProjectCard(project) {
-// //   const card = document.createElement('div');
-// //   card.classList.add('project-card');
-// //   card.innerHTML = `
-// //     <img src="${project.image}" alt="Project Image">
-// //     <h3>${project.name}</h3>
-// //     <button class="details-btn">Details</button>
-// //   `;
-// //   return card;
-// // }
+showCaseBtns.forEach((showCaseBtn) => {
+  // eslint-disable-next-line func-names
```

Comment:
- [ ] the linters are used to make your project work better and easier to read, therefor please avoid turning of any linters.